modules/ROOT/nav.adoc

```diff
@@ -358,6 +358,12 @@
  ** xref:xml/xml-xslt.adoc[XSLT Transformations with the XML Module]
```

Comment:
INSTRUCTIONS FOR EDIT:

Can you please edit this PR by EOD Monday? Please do NOT edit the zuora-aqua-connector-reference file (generated reference). The rest of the files are generated from the connector user guide template, and you have edited that content before. Very little of it is specific to this connector. The two files to concentrate on would be: zuora-aqua-connector-examples and zuora-aqua-connector.